src/__integrations__/pmClient.startSession.integration.test.ts

```diff
@@ -0,0 +1,168 @@
+    pmServerTerminator = createHttpTerminator({ server: pmServer });
+    // Start client
+    pmClient = createClient({
+      baseUrl: `http://${HOST}:${PORT}`,
+      fetchApi: retryingFetch(fetch, 5000 as Millisecond, 5),
+    });
+  });
+
+  afterAll(async () => {
+    await pmServerTerminator.terminate();
+  });
+
+  it('should return 500 when the request is empty', async () => {
+    expect(
+      (
+        await pmClient.startSessionUsingPOST({
+          startSessionRequest: {},
+        })
+      ).fold(
+        err => (err.pop()?.value as Response).status,
+        myRes => myRes.status,
+      ),
+    ).toEqual(500);
+  });
+
+  it('should return 422 when the request has an empty data field', async () => {
+    expect(
+      (
+        await pmClient.startSessionUsingPOST({
+          startSessionRequest: {
+            data: {},
+          },
+        })
+      ).fold(
+        err => (err.pop()?.value as Response).status,
+        myRes => myRes.status,
+      ),
+    ).toEqual(422);
+  });
+
+  it("should return the same response when it's called with or without device", async () => {
+    const pmResponse = await pmClient.startSessionUsingPOST({
+      startSessionRequest: {
+        data: {
+          email: 'nunzia.ross@example.com',
+          fiscalCode: 'UQNSFM56P12T733D', // seems to be ignored by PM
+          idPayment: '12345', // seems to be ignored by PM
+        },
+      },
+    });
+
+    const pmResponseWithDevice = await pmClient.startSessionUsingPOST({
+      startSessionRequest: {
+        data: {
+          email: 'nunzia.ross@example.com',
+          idPayment: '12345', // seems to be ignored by PM
+          device: {
+            os: 'ANDROID' as OsEnum,
+          },
+        },
+      },
+    });
+
+    expect(
+      pmResponse.fold(
+        () => undefined,
+        myRes => myRes.value?.data?.user,
+      ),
+    ).toEqual(
+      pmResponseWithDevice.fold(
+        () => undefined,
+        myRes => myRes.value?.data?.user,
+      ),
+    );
+  });
+
+  it('should return ANONYMOUS as user state when she feeds an email', async () => {
+    expect(
+      (
+        await pmClient.startSessionUsingPOST({
+          startSessionRequest: {
+            data: {
+              email: 'nunzia.ross@example.com',
+              fiscalCode: 'UQNSFM56P12T733D', // seems to be ignored by PM
+              idPayment: '12345', // seems to be ignored by PM
+            },
+          },
+        })
+      ).fold(
+        () => undefined,
+        myRes => myRes.value?.data?.user?.status,
+      ),
+    ).toEqual('ANONYMOUS');
+  });
+
+  it('should return a sessionToken when the user feeds an email', async () => {
+    const pmResponse = await pmClient.startSessionUsingPOST({
+      startSessionRequest: {
+        data: {
+          email: 'nunzia.ross@example.com',
+          fiscalCode: 'UQNSFM56P12T733D', // seems to be ignored by PM
+          idPayment: '12345', // seems to be ignored by PM
```

Comment:
the really required data for PM is the couple (`email` | `fiscal code` , `idPayment`), if all right the PM return a `sessionToken` and update the DB with one of previously data `email` | `fiscal code` related to `idPayment`

Reply:
@pasqualespica in the docker environment the PM returns a valid sessionToken even though I pass only the email